package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "private": true,
+  "name": "remnote-plugin-template-react",
```

Comment:
With the current version of RemNote plugins, it does seem everything is done in the manifest.json. I'll update that file to stay consistent just in case.

Reply:
No clue, but it wont hurt I guess